mitreattack/navlayers/README.md

```diff
@@ -249,3 +254,64 @@ conf.load_from_file(filename="path/to/poster/config.json")
+The form of this specific notation is that each technique has a score equal to the sum total of entities (Group, Software, or Mitigation) that use that technique, with the names of those relevant entities included in the technique's comment field.
+The module expects users to specify the domain and source during the initialization, and the type of entity whenever calling the generation method. 
+
+### OverviewGenerator()
+```python
+x = OverviewGenerator(source='taxii', domain='enterprise')
+```
+As stated above, the initialization for the generator expects the user to provide the domain and source. Valid options for domain include `enterprise`, `mobile`, and `ics`. 
+Valid options for source include `taxii`, and `local`, the latter of which requires the optional `local` parameter with a path to the local data store.
+
+### generate_layer()
+```python
+output_layer = x.generate_layer(obj_type='group')
+```
+This method creates and returns a ATT&CK Layer object (as defined within mitreattack-python core), with the appropriate notations for the given parameters. Valid `obj_type` values include `group`, `software`, and `mititagtion`.
+#### Example Usage
+```python
+from mitreattack.navlayers import OverviewGenerator
+
+x = OverviewGenerator(source='taxii', domain='mobile')
+output_layer_software = x.generate_layer(obj_type='software')
+output_layer_mitigation = x.generate_layer(obj_type='mitigation')
+
+output_layer_software.to_file(filename='demo_software_layer.json')
+print(output_layer_mitigation.to_str())
+```
+
+## usage_generator.py
+usage_generator provides the UsageGenerator class, which allows users to create either an Enterprise, Mobile, or ICS matrix containing notations on techniques corresponding to a specific Group, Software, or Mitigation. 
+The form of this notation is that each utilized technique will have a score of 1, and where the comment provided is the description of the entity. 
+The module expects users to specify the domain and source during the initialization, and to provide the specific entity when calling the generation method.
+
+### UsageGenerator()
+```python
+y = UsageGenerator(source='taxii', domain='enterprise')
+```
+As stated above, the initialization for the generator expects the user to provide the domain and source. Valid options for domain include `enterprise`, `mobile`, and `ics`. 
+Valid options for source include `taxii`, and `local`, the latter of which requires the optional `local` parameter with a path to the local data store.
+
+### generate_layer()
+```python
+y.generate_layer(match='APT29')
+```
+This method creates and returns a ATT&CK Layer object (as defined within mitreattack-python core), with the appropriate notations for the given parameters. Valid `match` values can be either a name, an alias/associated name, or ATT&CK ID.
```

Comment:
This is more consistent with how we refer to it in Workbench and the ATT&CK Website.

Website:
> The team makes a best effort to track overlaps between names based on publicly reported associations, which are designated as “Associated Groups” on each page (formerly labeled “Aliases”), because we believe these overlaps are useful for analyst awareness. 

Workbench:
> Overlaps between names based on publicly reported associations are tracked using "Associated Groups" (also known as "Aliases").

Reply:
Makes sense, even if alias is the "native" term.